test/parallel/test-stream-writableState-uncorked-bufferedRequestCount.js

```diff
@@ -7,7 +7,8 @@ const stream = require('stream');
 const writable = new stream.Writable();
 
 writable._writev = common.mustCall((chunks, cb) => {
-  assert.strictEqual(chunks.length, 2, 'two chunks to write');
+  //error: two chunks to write
```

Comment:
I think the comment can be removed. Maybe it's just me but I don't find it useful.

Reply:
I agree. This will also fail our linter due to the lack of a space after `//`.